x/oracle/keeper/owasm.go

```diff
@@ -22,10 +21,10 @@ func ConvertToOwasmGas(cosmos uint64) uint64 {
 
 // GetSpanSize return maximum value between MaxReportDataSize and MaxCallDataSize
 func (k Keeper) GetSpanSize(ctx sdk.Context) uint64 {
-	if k.MaxReportDataSize(ctx) > k.MaxCalldataSize(ctx) {
-		return k.MaxReportDataSize(ctx)
+	if k.GetParams(ctx).MaxReportDataSize > k.GetParams(ctx).MaxCalldataSize {
```

Comment:
Should we get params and store in variable first?

Reply:
ok